vmas/simulator/core.py

```diff
@@ -1045,11 +1045,14 @@ def render(self, env_index: int = 0) -> "List[Geom]":
         geoms = super().render(env_index)
         if len(geoms) == 0:
             return geoms
-        for geom in geoms:
-            geom.set_color(*self.color, alpha=self._alpha)
+
         if self._sensors is not None:
             for sensor in self._sensors:
                 geoms += sensor.render(env_index=env_index)
+
+        for geom in geoms:
+            geom.set_color(*self.color, alpha=self._alpha)
+
```

Comment:
What you are doing is now setting the colors of the sensor geoms to that of the agent